setup.py

```diff
@@ -39,8 +37,8 @@
         "resampy>=0.2.1,<0.3.0",
         #'scikit-image>=0.14.3,<0.15.0',
         "torch>=1.4.0",
-        "nnAudio",
-        "julius",
+        "nnAudio>=0.2.4",
+        "julius>=0.2.5",
```

Comment:
Does it work if you don't pin the version

Reply:
No. It doesn't. I checked